nemo/collections/llm/tools/autotuner/README.md

```diff
@@ -0,0 +1,421 @@
+# AutoTuner
+
+AutoTuner is a fully automated, predictive configuration builder and orchestrated tool that achieves maximum model throughput for LLM Training.
+
+## Overview
+
+AutoTuner extends NeMo's Auto Configurator with a fully automated pipeline that searches for hyperparameters achieving maximum training throughput while automatically preventing CUDA OOM errors and providing comprehensive performance & cost analysis for Large Language Models. Unlike research-only tools, AutoTuner runs directly on your GPU infrastructure, making it ready for production deployment and real-world training optimization.
+
+### Note
+
+AutoTuner is supported for all NeMo models including GPT-based models: GPT3, LLama, Mixtral, Mistral, Gemma, Nemotron, Starcoder, Qwen, and more.
+
+### AutoTuner Capabilities
+
+AutoTuner is designed to iterate over different model configurations quickly and find the best configuration that maximizes training throughput while minimizing both time and money. It offers comprehensive features to facilitate this, as detailed below.
+
+- **CUDA OOM Prevention**: Automatically detects and filters out configurations that would cause out-of-memory errors, preventing training failures
+- **Orchestrated Pipeline**: Fully automated workflow from configuration generation and pre-training to results analysis.
+- **Rigorous Performance Analysis**: Comprehensive metrics including TFLOPS, training time, and cost optimization.
+- **Rich Visualization**: Descriptive tables and charts for easy decision making and result interpretation.
+- **Production Deployment**: Runs directly on customer GPU infrastructure, ready for enterprise use.
+
+
+
+# NeMo Autotuner Launcher
+
+A tool for orchestrating NeMo autotuner workflows on remote clusters using Lepton. This launcher handles packaging local code and data, then executes autotuner steps remotely.
+
+## Quick Start
+
+### Prerequisites
+- Installation
+
+   ```bash
+   # Ensure you have NeMo, nemo_run and rich installed
+   pip install nemo nemo_run rich
```

Comment:
Is this the right `nemo` package? Isn't it `nemo_toolkit[nlp]`? Also, is there a wheel available on PyPI for `nemo_run` now, or does that still need to be installed via pointing at GitHub?